src/components/CohortBuilder/QueriesResolver.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Component from 'react-component-component';
+import graphql from 'services/arranger';
+
+const QueriesResolver = ({ children, queries, api }) => (
+  <Component
+    initialState={{ data: null, isLoading: true, error: null }}
+    didMount={({ setState }) => {
+      const { query, variables, transform = x => x } = queries[0];
```

Comment:
@ciaranschutte Why queries[0]?

Reply:
Oic, you will pass on all of the pie queries as an array,

Reply:
@nyanofthemoon Exactly. `queries[0]` to just have this one working.
It will be passed an array something like: 
``` javascript
[
  {
    query: `{
      file {
        aggregations {
          kf_id {
            buckets {
              key
            }
          }
        }
      }
    }`,
    variables: {
      sqon: sqon
    }
  },
  {
    query: `{
      file {
        aggregations {
          kf_id {
            buckets {
              key
            }
          }
        }
      }
    }`,
    variables: {
      sqon: sqon
    }
  },
```

This is part of https://app.zenhub.com/workspaces/oicr-sprints-59a846e9d6fb732a00631122/issues/kids-first/kf-portal-ui/1157

Reply:
yes, that's the idea :) all the queries will be batched into one single ajax request